pkg/scheduler/internal/queue/scheduling_queue.go

```diff
@@ -34,7 +34,7 @@ import (
 
 	"k8s.io/klog"
 
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
This is unnecessary, isn't it?

BTW: are you using vscode to format files upon saving?

Reply:
LOL yes. It seems that VSCode did that when I wasn't looking. Do you know if there's a setting for that?

Reply:
I met exactly the same issue and get really annoyed... You can try to change settings in `Go: Format Tool`, choose the "gofmt" option (I recalled it defaulted to "goreturns" somehow).